gradle/libs.versions.toml

```diff
@@ -28,17 +28,17 @@ androidx-dataStore = "1.0.0"
 androidx-exif = "1.3.6"
 androidx-jetpack = "1.1.0"
 androidx-lifecycle = "2.6.2"
-androidx-paging3 = "3.1.1"
-androidx-paging3Compose = "1.0.0-alpha18"
+androidx-paging3 = "3.2.1"
+androidx-paging3Compose = "3.2.1"
```

Comment:
question: is this necessary for the scope of this PR?

Reply:
I can create separate PR with this